### ISSUE TITLE
make subtitles button be a toggle

### ISSUE THREAD

**Issue by lollilol** [dyc3/opentogethertube#1001]
### Is your feature request related to a problem? Please describe.

When you click on the subtitles button, it asks you "on" or "off"

### Describe the solution you'd like

it should be a toggle, like it is on youtube

**Comment by dyc3** [comment]
Yeah this would probably be better. The problem is that if the video provides multiple captions tracks, it's impossible to know how many tracks there are for all players (particularly the youtube player). The On/Off toggle becomes a language selector when the available tracks become known.

I suppose we could just have it enable whatever track comes first when you click the button, and then subsequent clicks you bring up a list of all the tracks and an off button. Tbh, I'm not sure if that's better or worse.
